go.mod

```diff
@@ -172,4 +172,7 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace k8s.io/kubectl => k8s.io/kubectl v0.24.3
+replace (
+	github.com/open-traffic-generator/snappi/gosnappi => github.com/open-traffic-generator/snappi/gosnappi v0.9.8
+	k8s.io/kubectl => k8s.io/kubectl v0.24.3
+)
```

Comment:
for the first, it makes it explicit which version of gosnappi is used, otherwise if one of our dependencies upgrade gosnappi it will use the newest version.

for the second without it get 
```
go get -u github.com/openconfig/ondatra
go: reading k8s.io/kubectl/go.mod at revision v0.0.0: unknown revision v0.0.0
```

Reply:
I noticed this too look like a fine workaround.